### DISCUSSION TITLE
Ray tracing warning.

### DISCUSSION THREAD

**Discussion by claudiadom** [UCL/STIR#1403] [Q&A] [answered]
Hi I'm new to STIR,

Could someone explain me why is this warning showing:

`WARNING: ray tracing with equal start and end point. Returning zero`

Thanks beforehand.

**Comment by robbietuk** [comment, marked as answer]
The warning comes from here 
https://github.com/UCL/STIR/blob/8ced2d73933420457e0bc76074964b7d4ff00f0c/src/recon_buildblock/RayTraceVoxelsOnCartesianGrid.cxx#L51-L67

It appears to be an edge case in the ray tracer when the two coordinates are approximately the same
Could you expand on what STIR calls you are using and the parameters used to help understand the problem.
